### ISSUE TITLE
detect the background color of the plate

### ISSUE THREAD

**Issue by laplasz** [openalpr/openalpr#415]
Is it posible to detect the background color of the lincese plate?
in my country there is a meaning of each color, for example:
blue - diplomatic
yellow - taxi
green - zerro emission
white - regular

So would be great to detect the type of the plate by background color as well

**Comment by matthill** [comment]
OpenALPR doesn't do that out of the box, but it's possible.  The simplest way to do this would be to convert the plate area to HSV and use the hue (within a range) to determine what type of color the plate is.  Here's an example (used to detect flesh color from an image):
http://shervinemami.info/blobs.html

This won't work very well at night.

**Comment by laplasz** [comment]
Can you please point me where could I insert plate area detection? findOuterBoxMask does something like this? or maybe the plate area is hidden in the textdetection?
Thanks in advance!

**Comment by gstiebler** [comment]
Showing a histogram of hue values for different types of plates may help you determine the range of hue that should be used for each kind of plate.

**Comment by matthill** [comment]
Yes, you could do this after the plate has been recognized by OpenALPR.  We provide coordinates for the 4 corners of the plate.  So, you could grab the plate coordinates, crop out your plate from that, and run a hue analysis to get the color.
